mkdocs.yml

```diff
@@ -112,23 +112,29 @@ nav:
   - 'Getting Started':
       - 'Single-Cluster': getting-started-single-cluster.md
       - 'Multi-Cluster': getting-started-multi-cluster.md
-  - 'Architecture': architecture/docs/design/architectural-overview-v1.md
   - 'Installation':
       - 'Helm': https://artifacthub.io/packages/helm/kuadrant/kuadrant-operator
       - 'OLM on OpenShift': kuadrant-operator/doc/install/install-openshift.md
-  - 'Concepts and APIs':
+  - 'Concepts':
+      - 'Architecture': architecture/docs/design/architectural-overview-v1.md
       - 'DNSPolicy':
           - 'Overview': kuadrant-operator/doc/overviews/dns.md
-          - 'Reference': kuadrant-operator/doc/reference/dnspolicy.md
       - 'TLSPolicy':
           - 'Overview': kuadrant-operator/doc/overviews/tls.md
-          - 'Reference': kuadrant-operator/doc/reference/tlspolicy.md
       - 'AuthPolicy':
           - 'Overview': kuadrant-operator/doc/overviews/auth.md
-          - 'Reference': kuadrant-operator/doc/reference/authpolicy.md
       - 'RateLimitPolicy':
           - 'Overview': kuadrant-operator/doc/overviews/rate-limiting.md
-          - 'Reference': kuadrant-operator/doc/reference/ratelimitpolicy.md
+  - 'APIs':
+      - 'DNSPolicy':
+          - 'Reference': kuadrant-operator/doc/reference/dnspolicy.md
```

Comment:
What do you think of hoisting up the pages a level as there's only a single page below each nav here?
Same of 'Overview' pages above.

The nav would look something like this:

* Concepts
    * Architecture
    * DNSPolicy
    * TLSPolicy
    * ...
* APIs
    * DNSPolicy
    * TLSPolicy
    * ...

Reply:
sure that makes sense to me. I was thinking would there be anything in the future we write that might make sense with this structure say glossary for terminology for example but that can be decided then if it makes sense